docs/Components/AttachedComponent.md

```diff
@@ -60,7 +60,15 @@ plane.angle = math.radians(5)
 print(math.degrees(aileron.angle))
 >>> 14.99999999999
 ```
-## ambient property
+## position property
+This property returns the position vector of the component's origin in relation to its parent's origin
+```python
+wing.set_parent(plane)
+plane.angle = 8
+
+print(wing.position)
+>>>Vector2 (-0.17604664425338434, -0.17750374375522998)
+```## ambient property
 This property returns the environment in which the component is located
 ```python
 print(wing.ambient)
```

Comment:
The position docs got in the middle of the ambient docs.
Just move the line 63 to after the position docs